src/Ship.cpp

```diff
@@ -1035,22 +1035,16 @@ void Ship::TimeAccelAdjust(const float timeStep)
 	SetVelocity(GetVelocity() + vdiff);
 }
 
-double Ship::ExtrapolateHullTemperature() const
+double Ship::GetHullTemperature() const
 {
-	const double dragCoeff = DynamicBody::DEFAULT_DRAG_COEFF * 1.25;
 	// TODO: fix this to calculate appropriate skin friction and heating.
-	const double dragGs = CalcAtmosphericDrag(GetVelocity().LengthSqr(), GetClipRadius(), dragCoeff) / (GetMass() * 9.81);
+	//const double dragCoeff = DynamicBody::DEFAULT_DRAG_COEFF * 1.25;
+	//const double dragGs = CalcAtmosphericDrag(GetVelocity().LengthSqr(), GetClipRadius(), dragCoeff) / (GetMass() * 9.81);
 	//return dragGs / 25.0;
-	return 0.0;
-}
-
-double Ship::GetHullTemperature() const
-{
 	// TODO: fix this to properly account for heating due to air friction instead of G-force.
-	return 0.0;
 	double dragGs = GetAtmosForce().Length() / (GetMass() * 9.81);
 	int atmo_shield_cap = 0;
-	const_cast<Ship *>(this)->Properties().Get("atmo_shield_cap", atmo_shield_cap);
+	const_cast<Ship *>(this)->Properties().Get(R"(atmo_shield_cap)", atmo_shield_cap);
 	if (atmo_shield_cap && GetWheelState() < 1.0) {
 		return dragGs / (300.0 * atmo_shield_cap);
```

Comment:
This may be an existing issue, but even with the interim calculations, having an atmospheric shield equipped means you can do pretty much any maneuver in < 1Atm at any speed without consequences, and flying even an aerodynamic ship like the Wave without a shield results in near-instant death. I'd suggest changing the high multiplier from 300 to 150, and the low multiplier from 5 to 15. Some testing shows that these values should allow careful pilots to slowly fly ships without shielding, and make crazy maneuvers like deep aerobraking at 15+km/s more dangerous without a heavy shield.

Reply:
Ok, i'll try some tweaks